app/dao/usersDAO.py

```diff
@@ -30,3 +30,10 @@ def get_all_companies():
         return list(db["users"].find({"role":"company"},{"password":0}))
     except Exception as e:
         return e
+
+def get_approved_students():
+    try:
+        return list(db["users"].find({"role" : "student", "approved_by_college" : True},
```

Comment:
Only return fields required as per prototype

Reply:
Okay

Reply:
![c1](https://user-images.githubusercontent.com/51269966/81293077-568a1d00-908a-11ea-95f2-bd57c26d11b6.PNG)
These fields...?
Database don't have roll number, class & resume fields

Reply:
above details will be show when requesting for particular id....for approved students only return **_id, name, class, rollno, department**